### PR TITLE
Fix formatting in Add-Content.Tests.ps1 file.

### PR DESCRIPTION
**Pull request by sethvs**
## PR Summary

Fix formatting issues like absent spaces, noncapitalized cmdlets and skipped parameter names.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1 [hunk 1]
```diff
@@ -6,25 +6,25 @@ Describe "Add-Content cmdlet tests" -Tags "CI" {
 
   Context "Add-Content should actually add content" {
     It "should Add-Content to testdrive:\$file1" {
-      $result=add-content -path testdrive:\$file1 -value "ExpectedContent" -passthru
-      $result| Should -BeExactly "ExpectedContent"
+      $result = Add-Content -Path testdrive:\$file1 -Value "ExpectedContent" -PassThru
```
Comment (iSazonov) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1]: Please replace `testdrive:` with `$testdrive` in the file.

Comment (sethvs) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1]: about_TestDrive.help.txt specifies using TestDrive in this manner.

```
EXAMPLE
    function Add-Footer($path, $footer) {
        Add-Content $path -Value $footer
    }

    Describe "Add-Footer" {
        $testPath="TestDrive:\test.txt"
        Set-Content $testPath -value "my test text."
        Add-Footer $testPath "-Footer"
        $result = Get-Content $testPath

        It "adds a footer" {
            (-join $result).Should.Be("my test text.-Footer")
        }
    }
```

https://github.com/pester/Pester/blob/master/en-US/about_TestDrive.help.txt

Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1]: It is our common pattern. We use this in most files. This is more safe in some cases.
By the way unfortunately the Pester documentation is poorly updated.

Comment (sethvs) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1]: Pester documentation doesn’t have to follow our patterns of TestDrive usage. :)

I think, let’s codify this in some form, or just use it in accordance with common sense, like does it work in a specific case or not.

Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1]: I don't personally have a strong opinion on this.  Although it should at least be capitalized as `TestDrive:`.  cc @JamesWTruher

Comment (sethvs) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1]: Fixed.

**Comment by sethvs** [comment]
Travis CI failed with Save-Help.
Should I restart the tests somehow or just wait?

**Comment by adityapatwardhan** [comment]
@sethvs Restarted tests on Travis CI.

**Review by iSazonov — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**